packages/calling-component-bindings/src/utils/videoGalleryUtils.ts

```diff
@@ -77,6 +85,8 @@ export const convertRemoteParticipantToVideoGalleryRemoteParticipant = (
   userId: string,
   isMuted: boolean,
   isSpeaking: boolean,
+  /* @conditional-compile-remove(raise-hands) */
+  raisedHand: RaisedHand | undefined,
```

Comment:
```suggestion
  raisedHand?: RaisedHand,
```

We prefer ?: style instead of undefined